### PR TITLE
feat:  connect traveller flow to account and local storage.

### PR DESCRIPTION
**Pull request by Tulun**
1. Welcome traveller flow accessible from "Learn about Across" button.
2. Going through the flow sets localStorage variable based on connected account.

Screen Shot 2022-09-23 at 3 00 04 PM

Other PR will cover connecting this to backend data flows as needed.

### CODE REVIEW

File: src/views/PreLaunchAirdrop/components/TravellerFlow/TravellerFlow.tsx [hunk 1]
```diff
@@ -31,13 +39,20 @@ const TravellerFlow = () => {
...
+              setStep((pv) => Math.min(pv + 1, numDots));
+            }}
+            size="md"
+          >
+            Next
+          </SecondaryButtonV2>
+        ) : (
+          <ButtonV2 size="md" onClick={() => setActivePageFlow("splash")}>
+            Go to Bridge
+          </ButtonV2>
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/views/PreLaunchAirdrop/components/TravellerFlow/TravellerFlow.tsx:54]: Fair point, wasn't 100% sure.

File: src/views/PreLaunchAirdrop/components/SplashFlow/SplashFlow.tsx [hunk 2]
```diff
@@ -60,7 +63,16 @@ const SplashFlow = ({
...
-                  buttonHandler: () => {},
+                  buttonHandler: () => {
+                    // Check if user has seen traveller flow before
+                    // TODO: Connect to data flow in different PR.
+                    if (
+                      account &&
+                      !getAccountSeenWelcomeTravellerFlow(account)
+                    ) {
+                      setActivePageFlow("traveller");
+                    }
```
Comment (dohaki) [inline src/views/PreLaunchAirdrop/components/SplashFlow/SplashFlow.tsx:74]: I guess the flow will be defined in a different PR. But a user should still be able to view the traveller flow again even after clicking through no?

**Review by dohaki — APPROVED** [review]
LGTM. One comment that will get tackled in a different PR I guess.
